data/data/rhcos.json

```diff
@@ -53,8 +53,8 @@
         "image": "rhcos-43.80.20191002.1-azure.x86_64.vhd",
         "url": "https://rhcos.blob.core.windows.net/imagebucket/rhcos-43.80.20191002.1-azure.x86_64.vhd"
     },
-    "baseURI": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.80.20191002.1/x86_64/",
-    "buildid": "43.80.20191002.1",
+    "baseURI": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20191014.0/x86_64/",
```

Comment:
Will all of the platforms builds be available at this URI?  We use this location for IPI baremetal to download/cache the openstack qcow2 image so it'd be good to confirm it that's likely to work :)

Reply:
https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20191014.0/x86_64/rhcos-43.80.20191002.1-openstack.x86_64.qcow2 403s so I guess the answer is no - will we rename all the filenames to fcos to align with this?

Reply:
Yes, eventually. So far only Azure and GCP artifacts are not available.

I'm splitting updates into several PRs so that these could be tested and tracked separately

Reply:
These are xz compressed - https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20191014.0/x86_64/rhcos-43.80.20191002.1-openstack.x86_64.qcow2.xz, so for openstack/libvirt/baremetal a different PR is required, which implements xz uncompress

Reply:
Ack https://github.com/openshift/installer/pull/2657 merged which adds gz decompression and a way to plug in xz compression with a new decompress function.

We'll also need to update https://github.com/openshift/ironic-rhcos-downloader to support xz compression.

Reply:
Ah nice I see you have the installer xz support in-progress via https://github.com/openshift/installer/pull/2678 - I'll push a pr adding xz support to ironic-rhcos-downloader (which is also in the process of being renamed btw)

Reply:
Yeah, lets focus on vSphere in this PR. Other platforms support would be fixed in followup PRs.

Baremetal needs quite a few fixes:
* switch CI to replace 'ironic-*' images with metal3 images (so that we won't release OSP code in OKD)
* Use generic name in `ironic-rhcos-downloader` name (I don't think there is consensus on this yet)
* fix links in the installer and setup baremetal OKD CI